pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -432,6 +432,33 @@ def test_hash_equivalent(self):
         stamp = Timestamp(datetime(2011, 1, 1))
         assert d[stamp] == 5
 
+    @pytest.mark.parametrize(
+        "timezone, year, month, day, hour",
+        [["America/Chicago", 2013, 11, 3, 1], ["America/Santiago", 2021, 4, 3, 23]],
```

Comment:
is there anything special about these cases?  could just use tz_aware_fixture?

Reply:
The issue doesn't occur for all time zones or even all DST transitions. I'm not sure what sets these ones apart, but these DST transitions are cases where the current implementation seg faults.